### PR TITLE
add reset function and fix color circle ui

### PR DESCRIPTION
**Pull request by zhaoli1211**
use swift recommended setting 
added a reset initial color function
fix color circle too big issue

### CODE REVIEW

File: UNUMColorPicker/Classes/UNUMColorPickerViewController.swift [hunk 1]
```diff
@@ -93,3 +98,18 @@ extension UNUMColorPickerViewController: UICollectionViewDelegate {
         collectionView.reloadData()
     }
 }
+
+//MARK: UICol
```
Comment (plflanagan) [inline UNUMColorPicker/Classes/UNUMColorPickerViewController.swift]: Might as well fix this mark to `UICollectionViewDelegateFlowLayout`

File: UNUMColorPicker/Classes/UNUMColorPickerViewModel.swift [hunk 2]
```diff
@@ -11,7 +11,7 @@ import Foundation
 class UNUMColorPickerViewModel {
 
     let colors: [UIColor]
-    let initialColor: UIColor
+    var initialColor: UIColor
```
Comment (plflanagan) [inline UNUMColorPicker/Classes/UNUMColorPickerViewModel.swift:14]: why does this need to be a `var`? It's set in the initializer; when would it change?

File: UNUMColorPicker/Classes/UNUMColorPickerViewController.swift [hunk 3]
```diff
@@ -42,6 +42,11 @@ public class UNUMColorPickerViewController: UIViewController {
         self.viewModel = UNUMColorPickerViewModel(colors: colors, initialColor: initialColor)
     }
 
+    public func reset(initialColor color: UIColor) {
+        self.viewModel.initialColor = color
+        colorCollectionView.reloadData()
+    }
```
Comment (plflanagan) [inline UNUMColorPicker/Classes/UNUMColorPickerViewController.swift:48]: not sure i understand the use of this...

Comment (zhaoli1211) [reply UNUMColorPicker/Classes/UNUMColorPickerViewController.swift:48]: we can use this when we want to change other page's background color, so we don't need to create a new color picker view, but just reset the initial color of the existing color picker view

**Review by plflanagan — APPROVED** [review]
Not sure this is the right way to go or not; hard to tell when this isn't the use-case but is allowing for the use-case...

So, I'll say the code looks good for now, and see how the use-case looks when we get there.

**Pull request merged**